src/models/user.js

```diff
@@ -51,6 +51,9 @@ const userSchema = new mongoose.Schema(
     password: {
       type: String,
     },
+    passwordhash: {
+      type: String,
+    },
```

Comment:
This is not necessary as we are going to store password in hash. There is no need for extra passwordhash field.